results2junit

```diff
@@ -242,7 +242,7 @@ sub dump_junit {
             $test_suite_name, $class_name;
         printf $out_fh " name=\"%s\"", $test_name;
         printf $out_fh " time=\"%d\"",  $s->{run_time};
-        if ($s->{status} eq 'GOOD') {
+        if ($s->{status} eq 'GOOD' || $s->{status} eq 'TEST_NA') {
             print $out_fh "/>\n";
```

Comment:
dunno that we want TEST_NA == PASS.  It will confuse the idea of 'test shouldn't run here' with 'test was successful'.  In other words, if the test really starts running (and it shouldn't be) we won't notice (from the high-level results view)